core/src/main/java/org/apache/iceberg/puffin/StandardBlobTypes.java

```diff
@@ -26,4 +26,17 @@ private StandardBlobTypes() {}
    * href="https://datasketches.apache.org/">Apache DataSketches</a> library
    */
   public static final String APACHE_DATASKETCHES_THETA_V1 = "apache-datasketches-theta-v1";
+
+  /**
+   * A serialized form of <a
+   * href="https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/stats/StatsUtils.java#L854C17">
```

Comment:
nit: github links to a branch are likely to drift over time or even break (if eg module, package or class are renamed or removed)

Reply:
Done : Added a table in the hive wiki for supported column stats and replaced the github link to the wiki link.